### ISSUE TITLE
support rtl page progression direction

### ISSUE THREAD

**Issue by kmuto** [Trimmarks/versatype-issues#49]
vivliostyle-formatter's vertical writing support seems lack "page progression direction" support.

Pages of Japanese vertical writing should be placed as rtl (right->left->right->left...), but vivliostyle-formatter seems not have any options (command line or CSS) to handle such page progression direction, only ltr (left->right->left).

This problem affects spread view and "print on demand" systems. As far as I know Arabic uses rtl page direction also.

**Comment by kmuto** [comment]
In my test, set R2L to DIRECTION may solve it.

Here is a sample code implemented Java with using iText 2.1.7 (Debian stable, libitext-java package).

```
import java.util.*;
import com.lowagie.text.DocumentException;
import com.lowagie.text.pdf.*;
/* or com.itextpdf , depend on your iText version */

public class RtoL {
    public static void main(String args[]) {
        try {
            PdfReader reader = new PdfReader(args[0]);
            OutputStream os = new FileOutputStream(args[1]);
            PdfCopyFields copyFields = new PdfCopyFields(os);
            copyFields.addDocument(reader);
            copyFields.addViewerPreference(PdfName.DIRECTION, PdfName.R2L);
            copyFields.close();
            os.close();
        } catch (IOException e) {
            System.out.println(e);
        } catch (DocumentException e) {
            System.out.println(e);
        }
    }
}
```

**Comment by MurakamiShinyu** [comment]
This issue was moved to vivliostyle/vivliostyle.js#431
